src/opts/container.rs

```diff
@@ -331,6 +331,40 @@ impl ToString for PublishPort {
     }
 }
 
+pub enum IpcMode {
+    None,
+    Private,
+    Shareable,
+    Container(String),
+    Host,
+}
+
+impl ToString for IpcMode {
+    fn to_string(&self) -> String {
+        match &self {
+            IpcMode::None => String::from("none"),
+            IpcMode::Private => String::from("private"),
+            IpcMode::Shareable => String::from("shareable"),
+            IpcMode::Container(id) => format!("container:{}", id),
+            IpcMode::Host => String::from("host"),
+        }
+    }
+}
+
+pub enum PidMode {
+    Container(String),
+    Host,
+}
```

Comment:
Same here as with IpcMode.

Here is reference doc:
```
Set the PID (Process) Namespace mode for the container. It can be either:

    "container:<name|id>": joins another container's PID namespace
    "host": use the host's PID namespace inside the container


```